packages/grafana-data/src/types/icon.ts

```diff
@@ -245,6 +245,8 @@ export const availableIconsIndex = {
   'wrap-text': true,
   rss: true,
   x: true,
+  'add-user': true,
+  'clip-attach': true,
```

Comment:
i think `paperclip` is a better name, but 🤷

Reply:
I see you @joshhunt. Same thought about the "add-user" but it is named from the Unicon official library. 🤷🏻‍♂️

Reply:
hey folks 👋🏼 actually I just checked and the names of these icons in Unicon is `add-account` and `attach`. I think the names we are putting forward are the best ones for discoverability, but feel free if you'd like to rename - no big deal.

Reply:
I don't feel strongly about this (hence why I approved - you can merge whenever you're ready), but either paperclip or attach seem like the best options to me. add-user is fine to me.